padd.sh

```diff
@@ -108,6 +108,9 @@ padd_logo_retro_1="${bold_text} ${yellow_text}_${green_text}_      ${blue_text}_
 padd_logo_retro_2="${bold_text}${yellow_text}|${green_text}_${blue_text}_${cyan_text}) ${red_text}/${yellow_text}\\ ${blue_text}|  ${red_text}\\${yellow_text}|  ${cyan_text}\\  ${reset_text}"
 padd_logo_retro_3="${bold_text}${green_text}|   ${red_text}/${yellow_text}-${green_text}-${blue_text}\\${cyan_text}|${magenta_text}_${red_text}_${yellow_text}/${green_text}|${blue_text}_${cyan_text}_${magenta_text}/  ${reset_text}"
 
+# Positioning options: default position => top left (no centering)
```

Comment:
If we put those there, we should also move 

https://github.com/pi-hole/PADD/blob/30e7aa8d106652f6e7a5bc6607adf5412d6b26d6/padd.sh#L1261-L1267

up here.

Reply:
I think you're right.

Reply:
Actually, if we use `center` as the default option, this code will be useless.

By default, if no options are set the offsets will always be calculated.
If an option (`-xoff` or `-yoff`) is used, the value will also be set.
In any case, there will be no need to test if they are set.